Transformations/utilities/AddTimeStamps.py

```diff
@@ -0,0 +1,112 @@
+import logging
```

Comment:
Please precise in the beginning of the file the need for adding timestamps. Missing docstrings and comments for the functions.

Reply:
added precision in the [argparse description](https://github.com/VCityTeam/UD-Graph/blob/aa846ff115fd7ce7eb5a3348a11464d4a3b35351/Transformations/utilities/AddTimeStamps.py#L8) and the [readme](https://github.com/VCityTeam/UD-Graph/blob/rules/Transformations/utilities/README.md#addtimestamps)